tv.kodi.Kodi.json

```diff
@@ -0,0 +1,123 @@
+    "rename-icon": "kodi",
+    "modules": [
+        "shared-modules/udev/udev-175.json",
+        "shared-modules/python2.7/python-2.7.15.json",
+        "deps/avahi.json",
+        "deps/libical.json",
+        "deps/bluez.json",
+        "deps/fmt.json",
+        "deps/fribidi.json",
+        "deps/fstrcmp.json",
+        "deps/ghostscript.json",
+        "shared-modules/glu/glu-9.0.0.json",
+        "deps/libass.json",
+        "deps/libcdio.json",
+        "deps/lzo2.json",
+        "deps/samba.json",
+        "deps/swig.json",
+        "deps/taglib.json",
+        "deps/tinyxml.json",
+        {
+            "name": "kodi",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DVERBOSE=1",
+                "-DENABLE_INTERNAL_FFMPEG=ON",
+                "-DENABLE_INTERNAL_FLATBUFFERS=ON",
+                "-DENABLE_INTERNAL_RapidJSON=ON",
+                "-DFLATBUFFERS_URL=build/download/flatbuffers-1.9.0.tar.gz",
+                "-DCROSSGUID_URL=build/download/crossguid-8f399e8bd4.tar.gz",
+                "-DRapidJSON_URL=build/download/rapidjson-1.1.0.tar.gz",
+                "-DFFMPEG_URL=build/download/4.0.2-Leia-Alpha3.tar.gz",
+                "-DLIBDVDCSS_URL=build/download/1.4.1-Leia-Beta-3.tar.gz",
+                "-DLIBDVDREAD_URL=build/download/6.0.0-Leia-Alpha-3.tar.gz",
+                "-DLIBDVDNAV_URL=build/download/libdvdnav-6.0.0-Leia-Alpha-3.tar.gz",
+                "-DJava_JAVA_EXECUTABLE=/usr/lib/sdk/openjdk10/bin/java"
+            ],
+            "post-install": [
+                "sed -i 's|^KODI_DATA.*|KODI_DATA=${HOME}/.var/app/tv.kodi.Kodi/data|' /app/bin/kodi"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/xbmc/xbmc/archive/18.0b3-Leia.tar.gz",
+                    "sha256": "cc5f1a75287438b2336c49a265019a4cab9626235e05a70345d77e4cecd6dce3"
+                },
+                {
+                    "type": "patch",
+                    "path": "kodi-sh-in.patch"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "mkdir -p build/download"
+                    ]
+                },
+                {
+                    "type": "file",
+                    "url": "http://mirrors.kodi.tv/build-deps/sources/crossguid-8f399e8bd4.tar.gz",
+                    "sha256": "3d77d09a5df0de510aeeb940df4cb534787ddff3bb1828779753f5dfa1229d10",
+                    "dest-filename": "build/download/crossguid-8f399e8bd4.tar.gz"
+                },
+                {
+                    "type": "file",
+                    "url": "http://mirrors.kodi.tv/build-deps/sources/flatbuffers-1.9.0.tar.gz",
+                    "sha256": "5ca5491e4260cacae30f1a5786d109230db3f3a6e5a0eb45d0d0608293d247e3",
+                    "dest-filename": "build/download/flatbuffers-1.9.0.tar.gz"
+                },
```

Comment:
Build flatbuffers separate before. Look at pull request no. 3 in your repository.

Reply:
thanks, will pull this.